### ISSUE TITLE
Should we list Ethical Hacking YouTubers as well?

### ISSUE THREAD

**Issue by KendallDoesCoding** [collab-community/coding-youtubers#16]
Should we only list coding YouTubers or should we also list ethical hacking Youtubers? for ex: Loi Yang, zSecurity, Network Chuck?

Please let me know @collab-community/mantainers.

**Comment by Agrim-Sharma174** [comment]
Yes, good idea...
Can add, many folks look for these ones too

**Comment by KendallDoesCoding** [comment]
> Yes, good idea... Can add, many folks look for these ones too

👍

**Comment by fluentmoheshwar** [comment]
how about showing those in different pages
the link will be in the homepage.

**Comment by KendallDoesCoding** [comment]
> how about showing those in different pages the link will be in the homepage.

Yeah, agree with you on that. We may do that if the team is fine!

**Comment by KendallDoesCoding** [comment]
What do you think about this @vikasganiga05 @Adarsh-jaiss?

**Comment by vikasganiga05** [comment]
> What do you think about this @vikasganiga05 @Adarsh-jaiss?

Yes, It's a good idea. Most beginners are missing the security part in their development journey.

**Comment by fluentmoheshwar** [comment]
Do like #52
